src/pages/ShapeListPage/index.tsx

```
@@ -0,0 +1,44 @@
+  ShapeListHeader,
+  ShapeListWrapper,
+} from 'pages/ShapeListPage/ShapeListPage.styles';
+import ShapeListItem from 'pages/ShapeListPage/ShapeListItem';
+import SaveBtn from 'components/Button/SaveBtn';
+import Title from 'styles/TitleComponent';
+
+const ShapeListPage = () => {
+  const taskListRef = React.useRef<HTMLUListElement>(null);
+  const taskList = useAppSelector((state) => state.taskList.taskList);
+  const doneList = taskList.filter((task) => task.done === true);
+  const isDone = doneList.length > 0 ? false : true;
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)